README.md

```diff
@@ -202,6 +202,7 @@ or [GitLab examples](https://gitlab.com/UncleRus/esp-idf-lib/tree/master/example
 | **tsl2561**              | Driver for light-to-digital converter TSL2561                                    | BSD-3   | `esp32`, `esp8266`, `esp32s2`, `esp32c3` | Yes
 | **tsl2591**              | Driver for light-to-digital converter TSL2591                                    | MIT     | `esp32`, `esp8266`, `esp32s2`, `esp32c3` | Yes
 | **tsl4531**              | Driver for digital ambient light sensor TSL4531                                  | BSD-3   | `esp32`, `esp8266`, `esp32s2`, `esp32c3` | Yes
+| **veml7700**             | Driver for VEML7700 ambient light sensor                                         | ISC     | `esp32`, `esp8266`, `esp32s2`, `esp32c3` | Yes
```

Comment:
Please do not edit this file manually. See https://github.com/UncleRus/esp-idf-lib/blob/master/CONTRIBUTING.md#updating-readmemd

Reply:
And don't forget to add yourself to the credits list.